html/changelogs/Lohikar-NightmodeTweaks.yml

```diff
@@ -0,0 +1,40 @@
+################################
+# Example Changelog File
+#
+# Note: This file, and files beginning with ".", and files that don't end in ".yml" will not be read. If you change this file, you will look really dumb.
+#
+# Your changelog will be merged with a master changelog. (New stuff added only, and only on the date entry for the day it was merged.)
+# When it is, any changes listed below will disappear.
+#
+# Valid Prefixes: 
+#   bugfix
+#   wip (For works in progress)
+#   tweak
+#   soundadd
+#   sounddel
+#   rscadd (general adding of nice things)
+#   rscdel (general deleting of nice things)
+#   imageadd
+#   imagedel
+#   maptweak
+#   spellcheck (typo fixes)
+#   experiment
+#################################
+
+# Your name.  
+author: Lohikar
+
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - rscadd: "Added a night-mode control program for the Chief Engineer."
+  - tweak: "Changed how night-mode works internally."
+  - tweak: "Red alert now disables night-mode."
```

Comment:
It's good to see some technical work on it, although bedshaped was going to fix it anyways. I'm not sure it needed such a drastic overhaul

I'm not keen on red alert disabling it. thats a game balance thing, night mode can be very helpful to all kinds of antag activities, which are also the thing that causes red alert. Let people turn it off individually in that situation if ordered

I don't like restricting where it happens either. Night mode in the hallways, departures, etc has proven to be far more interesting and immersive. I don't see a reason to restrict it from anywhere, if someone needs more light in their lab they can toggle it manually.

It's a no from me on the design decisions here, although ill leave it to bedshaped to address the technical specifics